keyboards/crkbd/keymaps/rustyjonez/keymap.c

```diff
@@ -0,0 +1,343 @@
+     KC_TAB, KC_A, KC_S, KC_D, KC_F, KC_G,                                    KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT,\
+     KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B,                                   KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_ENT,\
+                                    KC_ESC, KC_LCTL, KC_SPC,                  KC_SPC, KC_NO, TG(_GAME)\
+    ),
+
+  [_SYMBOL] = LAYOUT( \
+     KC_TILD, KC_EXLM, KC_AT, KC_HASH, KC_DLR, KC_PERC,                        KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_NO,\
+     KC_LSFT, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN,                    KC_NO, KC_DLR, KC_PERC, KC_CIRC, KC_UNDS, KC_PLUS,\
+     KC_LCTL, KC_PSLS, KC_PIPE, KC_BSLS, KC_LBRC, KC_RBRC,                      KC_NO, KC_EXLM, KC_AT, KC_HASH, KC_QUES, KC_NO,\
+                                    KC_NO, KC_LCBR, KC_RCBR,                      KC_NO, KC_TRNS, KC_NO\
+    )
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    if (record->event.pressed) {
+#ifdef OLED_DRIVER_ENABLE
+        oled_timer = timer_read32();
+        add_keylog(keycode);
+#endif
+#ifndef SPLIT_KEYBOARD
+        if (keycode == RESET && !is_master) {
+            return false;
+        }
+#endif
+    }
+    return true;
+}
+
+ #ifdef OLED_DRIVER_ENABLE
+oled_rotation_t oled_init_user(oled_rotation_t rotation) {
+  if (is_master) {
+    return OLED_ROTATION_270;
+  } else {
+    return OLED_ROTATION_180;
+  }
+}
+
+void add_keylog(uint16_t keycode) {
+    if ((keycode >= QK_MOD_TAP && keycode <= QK_MOD_TAP_MAX) || (keycode >= QK_LAYER_TAP && keycode <= QK_LAYER_TAP_MAX) || (keycode >= QK_MODS && keycode <= QK_MODS_MAX)) {
+        keycode = keycode & 0xFF;
+    } else if (keycode > 0xFF) {
+        keycode = 0;
+    }
+
+    for (uint8_t i = 4; i > 0; --i) {
+        keylog_str[i] = keylog_str[i - 1];
+    }
+
+    if (keycode < (sizeof(code_to_name) / sizeof(char))) {
+        keylog_str[0] = pgm_read_byte(&code_to_name[keycode]);
+    }
+
+    log_timer = timer_read();
+}
+
+void update_log(void) {
+    if (timer_elapsed(log_timer) > 750) {
+        //add_keylog(0);
+    }
+}
+
+void render_keylogger_status(void) {
+    oled_write_P(PSTR("KLGGR"), false);
+    oled_write(keylog_str, false);
+}
+
+
+void render_layer_state(void) {
+    oled_write_ln_P(PSTR("LAYER"), false);
+    oled_write_ln_P(PSTR("NUM"), layer_state_is(_NUM));
+    oled_write_ln_P(PSTR("NAV"), layer_state_is(_NAV));
+    oled_write_ln_P(PSTR("FUNC"), layer_state_is(_FUNCT));
+    oled_write_ln_P(PSTR("GAME"), layer_state_is(_GAME));
+    oled_write_ln_P(PSTR("SYMBL"), layer_state_is(_SYMBOL));
+}
+
+void render_keylock_status(uint8_t led_usb_state) {
+    oled_write_P(PSTR("Lock:"), false);
+    oled_write_P(PSTR(" "), false);
+    oled_write_P(PSTR("N"), led_usb_state & (1 << USB_LED_NUM_LOCK));
+    oled_write_P(PSTR("C"), led_usb_state & (1 << USB_LED_CAPS_LOCK));
+    oled_write_ln_P(PSTR("S"), led_usb_state & (1 << USB_LED_SCROLL_LOCK));
```

Comment:
```suggestion
    oled_write_ln_P(PSTR("S"), IS_LED_ON(led_usb_state, USB_LED_SCROLL_LOCK));
```

Reply:
I implemented all of these to us the IS_LED_ON per your suggestion, thank you!